infrastructure/helm/tigerbeetle/values.yaml

```diff
@@ -23,10 +23,10 @@ statefulset:
   clusterId: "0"
 
   image:
-    repository: ghcr.io/tigerbeetledb/tigerbeetle@sha256
+    repository: ghcr.io/tigerbeetledb/tigerbeetle
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: "621268959801084246679f25416a9451f553fe3697dd4f618f39503322079433"
+    tag: "latest"
```

Comment:
I think these should stay pinned buy I'm having trouble finding the hash for the commit
https://github.com/tigerbeetledb/tigerbeetle/commit/dfa2c14a997a9fc9b95d5c7d7e387e938a2786fe
https://github.com/tigerbeetledb/tigerbeetle/pkgs/container/tigerbeetle/versions?filters%5Bversion_type%5D=untagged

Reply:
<img width="745" alt="image" src="https://user-images.githubusercontent.com/16089022/222248853-f4bc2903-74ae-4475-8b14-29dd18c492b8.png">
click on the three dots and it is revealed.

Reply:
But that `latest` doesn't correspond to the tigerbeetle-node v0.12.0 commit

Reply:
No, it's the tigerbeetle db image hash. I don't know if there is a way of referencing the commit hash and the image hash.

Reply:
I've previously found the image hash in the Docker workflow job for the `tigerbeetledb/tigerbeetle` commit in which the node client's package.json version is bumped.
But I can't find it for that job for this commit

Reply:
I think I found it: https://github.com/tigerbeetledb/tigerbeetle/actions/runs/4282463687/jobs/7456772344

However, their actions are failing. Should we use the hash of the first push to main where the actions aren't failing anymore?

Reply:
@wilsonianb 👆

Reply:
I think that's from several commits later.
It _might_ still work, but it'd be pairing the node client with a different version of tigerbeetle

Reply:
What I linked was the action referring to the commit where the node client version in the package.json changed.